pkgs/development/web/postman/default.nix

```diff
@@ -84,9 +117,7 @@ stdenv.mkDerivation rec {
     mkdir -p $iconSizeDir
     ln -s $out/share/postman/resources/app/assets/icon.png $iconRootDir/postman.png
     ln -s $out/share/postman/resources/app/assets/icon.png $iconSizeDir/postman.png
-  '';
 
-  postFixup = ''
```

Comment:
That was not what I meant.
1. Split the two phases again.
2. Add the following to `installPhase`:
```nix
installPhase = ''
  runHook preInstall
  ....
  runHook postInstall
'';
```

Reply:
Ah, I see. That makes sense. Done in c9f600133359df813f45da4fdd413d4392caedfd .